File: DeviceServer/Models/ChangeSettings.cs
namespace DeviceServer.Models;

public class ChangeSettings
{
    public double MaxHumidity { get; set; }
    public double MinHumidity { get; set; }
    public double MaxBrightness { get; set; }
    public double MinBrightness { get; set; }
    public string BrightPeriodStart { get; set; }
    public string BrightPeriodEnd { get; set; }
}

File: User_Server_BH_basic/app/models/change_settings.py
from pydantic import BaseModel, Field
from datetime import datetime, time

class ChangeSettingsCreate(BaseModel):
    max_humidity: float = Field(..., alias="maxHumidity")
    min_humidity: float = Field(..., alias="minHumidity")
    max_brightness: float = Field(..., alias="maxBrightness")
    min_brightness: float = Field(..., alias="minBrightness")
    bright_period_start: time = Field(..., alias="brightPeriodStart")
    bright_period_end: time = Field(..., alias="brightPeriodEnd")
    device_id: int = Field(..., alias="deviceId")

    class Config:
        allow_population_by_field_name = True

class ChangeSettingsOutput(BaseModel):
    result: str
    updated_at: datetime